forms-flow-bpm/src/test/java/org/camunda/bpm/extension/hooks/delegates/FormTextAnalysisDelegateTest.java

```diff
@@ -55,10 +60,12 @@ public void setup() {
      * This test case perform a positive test over execute method in FormTextAnalysisDelegate
      * This will verify the textSentimentRequest
      */
-   /* @Test
+    @Test
```

Comment:
why there is a code change. please revert this

Reply:
Reverted changes